src/it/simple-it/src/main/resources/databricks-plugin/databricks-job-settings.json

```diff
@@ -22,14 +22,17 @@
       "enable_elastic_disk": false
     },
     "email_notifications": {
+      "on_failure": [
+        "fake@fake.com"
```

Comment:
Very minor: AFAIK, this is a valid email address. Consider using something that will never resolve, like "fake@example.com".

Reply:
Oh interesting. I'll change it next time.